src/services/outliningElementsCollector.ts

```diff
@@ -6,9 +6,14 @@ namespace ts.OutliningElementsCollector {
     export function collectElements(sourceFile: SourceFile, cancellationToken: CancellationToken): OutliningSpan[] {
         const elements: OutliningSpan[] = [];
         let depth = 0;
+        const regions: RegionRange[] = [];
+        const regionText = "#region";

```

Comment:
Maybe rename to `regionDelimiter` or `regionStartDelimter`? It's a bit confusing when reading `isRegionStart` below.